### PR TITLE
Follow up on FTBFS bugs.

### LINKED ISSUE #52: Follow up on FTBFS tickets
It's a similar workflow to what hotness does now with upstream release tickets, but it would be cool if it also followed up on ftbfs tickets when something finally starts getting built again in rawhide.

### PR DESCRIPTION
**Pull request by ralphbean**
Fixes #52.

Tested with partner-bugzilla, see https://partner-bugzilla.redhat.com/show_bug.cgi?id=1105934

### CODE REVIEW

File: hotness/bz.py [hunk 1]
```diff
@@ -126,7 +126,28 @@ def attach_patch(self, filename, description, bug):
                 is_patch=True)
         self.log.info("Attached patch to bug: %s" % bug.weburl)
 
+    def ftbfs_bug(self, name):
+        """ Return the first FTBFS bug we find for a package """
+        short_desc_pattern = '%s: FTBFS in rawhide' % name
+        query = {
+            'component': name,
+            'bug_status': self.bug_status_open + self.bug_status_closed,
```
Comment (pypingou) [inline hotness/bz.py]: Do we need closed bugs?

File: hotness/bz.py [hunk 2]
```diff
@@ -126,7 +126,28 @@ def attach_patch(self, filename, description, bug):
...
+            'query_format': 'advanced',
+        }
+        bugs = self.bugzilla.query(query)
+        bugs = bugs or []
+        for bug in bugs:
+            # The short_desc_pattern contains a space at the end, which is
+            # currently not recognized by bugzilla. Therefore this test is
+            # required:
+            if bug.short_desc.startswith(short_desc_pattern):
+                return bug
```
Comment (pypingou) [inline hotness/bz.py]: Should we yield instead of return here?

**Comment by pypingou** [comment]
1 question, otherwise :+1: for me

**Comment by ralphbean** [comment]
@pypingou good points made.  I pushed two new commits to address them.

**Comment by pypingou** [comment]
:+1: for me

**Pull request merged**